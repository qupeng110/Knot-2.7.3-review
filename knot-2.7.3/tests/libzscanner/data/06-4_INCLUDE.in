$ORIGIN .
$TTL	1

; KO
$INCLUDE @TMPDIR@/file-doesnt-exist		; File doesn't exist
