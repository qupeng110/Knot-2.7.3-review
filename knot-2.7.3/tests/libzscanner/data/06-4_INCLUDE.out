ERROR=ZS_FILE_OPEN
------
